scoring.js

```diff
@@ -0,0 +1,131 @@
+// Data function for Tightend
+function CalcScoreunknown(Player) {
+  let result = 0
+
+  return result
+}
```

Comment:
For this function's return statement, you could simply say "return 0" to shorten it a bit